src/magicbox/ResultPreviewsManager.ts

```diff
@@ -194,17 +196,21 @@ export class ResultPreviewsManager {
     return optionsEventArgs;
   }
 
-  private getSearchResultPreviewsQuery(suggestion: HTMLElement) {
+  private getSearchResultPreviewsQuery(suggestion: Suggestion) {
     const populateEventArgs: IPopulateSearchResultPreviewsEventArgs = {
-      suggestionText: suggestion.innerText,
+      suggestion,
       previewsQueries: []
     };
     $$(this.root).trigger(ResultPreviewsManagerEvents.populateSearchResultPreviews, populateEventArgs);
     return this.previewsProcessor.processQueries(populateEventArgs.previewsQueries);
   }
 
-  private updateSearchResultPreviewsHeader(suggestion: string) {
-    this.resultPreviewsHeader.text(`${this.options.previewHeaderText} "${suggestion}"`);
+  private updateSearchResultPreviewsHeader(suggestion: Suggestion) {
+    let text = `${this.options.previewHeaderText} "${suggestion.text}"`;
+    if (suggestion.field) {
+      text += ` ${this.options.previewHeaderFieldText} "${suggestion.field.value}"`;
```

Comment:
Why is this needed ?
 Can you show a screenshot of what it produces ? 
I'm fairly certain this will create weird random headers with some setup.

Reply:
<img width="923" alt="Screen Shot 2019-12-18 at 6 05 50 PM" src="https://user-images.githubusercontent.com/4923043/71130910-1e5c6d80-21c1-11ea-87d3-b1f7f09e5d6f.png">
Works quite well

Reply:
Let's see if DOC agrees though 😄

Reply:
TBH, I'm not a huge fan of the current message :)

Technically, these aren't necessarily _recommendations_. They're the top results you would get when actually executing the previewed suggested query. In fact, those results may not even be boosted by ART.

Also, while the component works well for commerce, it is not altogether unthinkable that it could be used in a different context. Thus, we would not necessarily be showing "products" in the query suggestion preview.

Finally, there's a new ML Product Recommendations model that works in a similar fashion to Event Recommendations. I'm concerned that branding  _query suggestion previews_ as _product recommendations_ could be slightly confusing for end users (and even for developers working on a solution).

How about something along those lines:

> Most relevant results for "query" [in "CATEGORY"]

(I'd use "in" rather than "under" for the sake of uniformity; this is what we're using by default when displaying facet value suggestions)

Reply:
Good idea, I'll change those strings. As for @olamothe's question on if this is needed, I added it to clarify that those results belong to a facet, but indeed it could be ommited. For now, I'll follow @fbeaudoincoveo's advice until more comments come in.

Reply:
Good point, I'll ask UX and link them this conversation.

Reply:
Can we take the simplest approach ?

If we have this header, then I'm willing to bet that in the near future, someone will ask us to change it/modify it/customize it/have an option for it.

Because, as a end user, I know if I saw that header, I would definitely want to change it. The wording will **always** be weird/unfit for some scenarios.

We also need to think about localization.

And so even more complexity !

Versus if we just remove the header entirely... then to me it seems like users will still understand that they are seeing "previews" for the current suggestions. 

We don't need to have a header that repeat that.

Reply:
Another important sidenote: if we do include the facet value in the status, should we include every facet values? It appears that multiple facet values can be received with `isCategoryField`.

Reply:
Hello :)
I would remove the header completly to reduce complexity like Oli said and also because I don't think it is necessary to understand what you are seeing. 
Other thing to add : I suggest to not put the "in FITBIT" in caps in the query suggest. In a different color is enough to make the disctinction. 
So we would read : **Fitbit in** Batteries (with the _in Batteries_ in gray).

Reply:
Sounds good! I'll remove the header altogether.